File: problems/0058/0058.cpp
#include <string>

class Solution {
public:
    int lengthOfLastWord(std::string s) {
        int start = s.size() - 1;
        while (s[start] == ' ') {
            --start;
        }

        int cnt = 0;
        for (int i = start; i >= 0; --i) {
            if (s[i] == ' ') {
                return cnt;
            }
            ++cnt;
        }
        return cnt;
    }
};

File: problems/0001 - 0400/0058/0058.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        start = len(s) - 1
        while s[start] == ' ':
            start -= 1
        cnt = 0
        for i in range(start, -1, -1):
            if s[i] == ' ':
                return cnt
            cnt += 1
        return cnt
